trieTools/trieToolsCommon/common.go

```diff
@@ -146,12 +149,33 @@ func GetTrie(flags ContextFlagsConfig, maxDBValue int) (common.Trie, error) {
 		EnabledDbLookupExtensions: false,
 	}
 
-	db, err := pruning.NewTriePruningStorer(args)
-	if err != nil {
-		return nil, err
+	return pruning.NewTriePruningStorer(args)
+}
+
+// GetStorer will create and return a storer using the provided flags
+func GetStorer(flags ContextFlagsConfig) (storage.Storer, error) {
+	localDbConfig := dbConfig // copy
+	localDbConfig.FilePath = path.Join(flags.WorkingDir, flags.DbDir)
+	dbPath := path.Join(flags.WorkingDir, flags.DbDir)
+
+	dbConf := storageUnit.DBConfig{
+		FilePath:          dbPath,
+		Type:              storageUnit.DBType(dbConfig.Type),
+		BatchDelaySeconds: dbConfig.BatchDelaySeconds,
+		MaxBatchSize:      dbConfig.MaxBatchSize,
+		MaxOpenFiles:      dbConfig.MaxOpenFiles,
+	}
+
+	return storageUnit.NewStorageUnitFromConf(cacheConfig, dbConf)
+}
+
+// GetTrie will create and return a trie using the provided flags
+func GetTrie(storer storage.Storer) (common.Trie, error) {
```

Comment:
`GetTrie()` and `GetStorer()` create objects, but that should be all right in this context.

Reply:
Renamed them to Create.... as they better suggest what are they doing. Thanks